crates/holochain_types/src/app.rs

```diff
@@ -996,6 +996,8 @@ pub enum DisabledAppReason {
     NeverStarted,
     /// The disabling was done manually by the user (via admin interface)
     User,
+    /// Disabling app in order to delete its agent key and render all chains read-only.
+    DeleteAgentKey,
```

Comment:
This sounds more like a status name:

```suggestion
    NoValidAgentKey,
```

Reply:
Same with the doc comment, please can it describe a state rather than an action?

Reply:
Gentledevs, why a state/status? The enum is called `DisabledAppReason` and the doc comment says the _reason_ an app is in a disabled state. `User` and `Error` don't sound like states.

Reply:
Maybe the other variants are not statuses, but since the enum is called *Reason, I think the variant should at least sound like a reason. "app disabled due to the user, due to an error, due to it never being started, due to there being no valid agent key", those all rhyme.